projects/marketing-site/public/lang/common.json

```diff
@@ -0,0 +1,242 @@
+{
+    "language": [
+        {
+            "locale": "en-US",
+            "name": "English"
+        },
+        {
+            "locale": "en-US",
+            "name": "Vietnamese"
+        },
+        {
+            "locale": "vi-VN",
+            "name": "English"
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Tiếng Việt"
+        }
+    ],
+    "menu": [
+        {
+            "locale": "en-US",
+            "vision": "Vision",
+            "team": "Team",
+            "funding": "Funding",
+            "token": "Token",
+            "language": "Language",
+            "dark_mode": "Dark Mode",
+            "sign_out": "Sign out"
+        },
+        {
+            "locale": "vi-VN",
+            "vision": "Tầm nhìn",
+            "team": "Đội ngũ",
+            "funding": "Kinh phí",
+            "token": "Token",
```

Comment:
is the word for Token the same?

Reply:
Its the same, in vietnam we use word "token" can't translate